src/Sound/AudioDebug.cpp

```diff
@@ -0,0 +1,287 @@
+/*****************************************************************************
+ * Copyright (c) 2018-2020 openblack developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/openblack/openblack
+ *
+ * openblack is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#include "AudioDebug.h"
+#include "SoundHandler.h"
+#include <imgui.h>
+
+using namespace openblack;
+using namespace openblack::audio;
+
+PlayType AudioDebug::_playType = PlayType::Once;
+SoundId AudioDebug::_selectedSound = -1;
+std::string AudioDebug::_selectedPack = "";
+AudioEmitterId AudioDebug::_selectedEmitter = InitialAudioEmitterId;
+const ImVec4 redColor = ImVec4(1.f, .0f, .0f, 1.f);
+const ImVec4 greenColor = ImVec4(0.f, 1.f, .0f, 1.f);
+
+void AudioDebug::AudioPlayer(Game& game, const std::vector<std::shared_ptr<SoundPack>>& soundPacks)
+{
+	auto& handler = game.GetSoundHandler();
+	if (ImGui::Button("Play") && _selectedSound != -1)
```

Comment:
error: comparison of integer expressions of different signedness: ‘openblack::audio::SoundId’ {aka ‘long unsigned int’} and ‘int’ [-Werror=sign-compare]